lms/templates/dashboard/_dashboard_credit_info.html

```diff
@@ -21,10 +21,7 @@
         </p>
     % if not credit_status["purchased"] and not credit_status["error"] :
         <p class="message-copy credit-eligibility-msg">
-            ${_("You are now eligible for credit. {congrats}").format(
-                    congrats="<b>Congratulations</b>",
-                )
-            }
+            ${_("You are now eligible for credit. <b>Congratulations!</b>")}
```

Comment:
@zubair-arbi It's a toss up. I tend to leave small bits of html in the string when necessary (like `<b> </b>`). The reason we take the <a href> out is because it's really long (`<a href='some-long-url-that-would-be-bad-if-translators-changed'>`). It's better to remove all html when possible, for example if the whole phrase was bolded I would suggest doing

```
"<b>" + _("You are now eligible for credit. Congratulations!") + "</b>"
```

Unfortunately that's not the case here.